.ci/packer_cache.sh

```diff
@@ -16,12 +16,21 @@ while [ -h "$SCRIPT" ] ; do
 done
 
 source $(dirname "${SCRIPT}")/java-versions.properties
-export JAVA_HOME="${HOME}"/.java/${ES_BUILD_JAVA}
-# We are caching BWC versions too, need these so we can build those
+## We are caching BWC versions too, need these so we can build those
 export JAVA8_HOME="${HOME}"/.java/java8
 export JAVA11_HOME="${HOME}"/.java/java11
 export JAVA12_HOME="${HOME}"/.java/openjdk12
 export JAVA13_HOME="${HOME}"/.java/openjdk13
 export JAVA14_HOME="${HOME}"/.java/openjdk14
-./gradlew --parallel clean -s resolveAllDependencies
 
+## 6.8 branch is not referenced from any bwc project in master so we need to
+## resolve its dependencies explicitly
+rm -rf checkout/6.8
+git clone --reference $(dirname "${SCRIPT}")/../.git https://github.com/elastic/elasticsearch.git --branch 6.8 --single-branch checkout/6.8
+export JAVA_HOME="${JAVA11_HOME}"
+./checkout/6.8/gradlew --project-dir ./checkout/6.8 --parallel clean --scan -Porg.elasticsearch.acceptScanTOS=true --stacktrace resolveAllDependencies
+
```

Comment:
Maybe `rm -rf checkout/6.8` just to keep the build image a little smaller.

Reply:
added this